syntax = "proto3";

package User;

message TPhoto {
  enum EPhotoType {
    EPT_UNKNOWN = 0;
    EPT_JPG = 1;
  }
  EPhotoType Type = 1;
  bytes Data = 2;
}

message TUserKey {
  uint64 Hash = 1;
}

message TUser {
  TUserKey Key = 1;
  string Name = 2;
  string Descripton = 3;
  repeated TPhoto Photos = 4;
}
